client/components/link-card/index.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from '@emotion/styled';
+import { ReactChild } from 'react';
+
+import './style.scss';
+
+interface LinkCardContainerProps {
+	background: string;
+}
+
+interface LinkCardProps {
+	label: ReactChild;
+	title: ReactChild;
+	cta: ReactChild;
+	background: string;
+	url: string;
```

Comment:
Is it worth spending a little bit of time making all these optional? Thinking it might help re-usability.